### PR TITLE
adding interpretation function examples to docs

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
@mattansb Before I work on the rest of the functions, is this what you had in mind?

File: R/cohens_d.R [hunk 1]
```diff
@@ -47,23 +47,45 @@
...
+#' interpret_d(0.4, rules = "cohen1988")
+#' interpret_g(0.4, rules = "sawilowsky2009")
+#' interpret_delta(0.4, rules = "gignac2016")
+#'
 #' @references
-#' - Cohen, J. (1988). Statistical power analysis for the behavioral sciences (2nd Ed.). New York: Routledge.
-#' - Hedges, L. V. & Olkin, I. (1985). Statistical methods for meta-analysis. Orlando, FL: Academic Press.
-#' - Hunter, J. E., & Schmidt, F. L. (2004). Methods of meta-analysis: Correcting error and bias in research findings. Sage.
-#' - McGrath, R. E., & Meyer, G. J. (2006). When effect sizes disagree: the case of r and d. Psychological methods, 11(4), 386.
+#' \itemize {
```
Comment (mattansb) [inline R/cohens_d.R:79]: Why is not not in md???

**Comment by mattansb** [comment]
Yup, exactly (:

Note tests are failing tho for some reason due to formatting?

File: R/cohens_d.R (hunk 1 above)
Comment (IndrajeetPatil) [reply R/cohens_d.R]: Oh, I didn't realize we are using `md` throughout `easystats` now. I have reverted this change.

**Comment by IndrajeetPatil** [comment]
Yupp, fixed the failing examples.

Cool, I will work on the rest of the functions then.

**Comment by IndrajeetPatil** [comment]
Does `glass_delta` not support one-sample tests?

**Comment by mattansb** [comment]
Nope - needs two samples by definition (:

**Comment by IndrajeetPatil** [comment]
But then shouldn't the function definition make that clear? That is, `y = NULL` should be changed to `y`

https://github.com/easystats/effectsize/blob/c7100c96a0863ac873a418b9c5020591da2c7907/R/cohens_d.R#L160

I can make that change in this PR if you want.

**Comment by IndrajeetPatil** [comment]
Oh, I see. Never mind. I get it, since `x` can also be specified as a formula, `y` can be `NULL`.

**Review by mattansb — APPROVED** [review]
Look good.

I just hope the formatting of the itemized docs doesn't break anything (:

**Comment by IndrajeetPatil** [comment]
> I just hope the formatting of the itemized docs doesn't break anything (:

It shouldn't. I have checked how the docs look in the `help` section of `RStudio` IDE, and they look okay. So they should also look okay on the website.

**Comment by mattansb** [comment]
Cool - thanks!

**Pull request merged**